doc/source/cluster/key-concepts.rst

```diff
@@ -60,19 +60,31 @@ sit idle, the autoscaler will remove worker nodes from the cluster.
 It is important to understand that the autoscaler only reacts to task and actor resource requests, and not application metrics or physical resource utilization.
 To learn more about autoscaling, refer to the user guides for Ray clusters on :ref:`VMs <cloud-vm-index>` and :ref:`Kubernetes <kuberay-index>`.
 
+
+.. _ray_driver:
+
+Ray Driver
+----------
+
+A Ray driver is the process that calls the Ray tasks and actors in the user application.
+A driver is started by running `ray.init()`. The script that calls `ray.init()` may also be referred to as the "driver script".
+
+
 .. _cluster-clients-and-jobs:
 
-Ray Jobs
---------
+Ray Job
+-------
+
+A Ray Job is a user application that includes a Ray driver. There are two ways of running Ray Jobs on Ray clusters:
```

Comment:
I am a little confused about this part. I thought we decided to call both driver & submission job as "jobs"? cc @scottsun94

Reply:
That's what was intended here.  Or are you talking about the fact that it says "Ray Job" instead of "job"?

Reply:
Submission jobs also contain a ray driver.  Technically they don't have to (the entrypoint could just be `echo hello` instead of `python my_ray_script.py`), but it seems like an edge case and I'd rather not confuse the reader here

Reply:
I feel @scottsun94's understanding is more clear:

Submission is not a type of job, it's just one way to run the job. so we only have one Job concept but multiple ways to run it.

Reply:
I think I am happy with Stephanie's definition. Btw isn't this 3 ways, not 2 ways? (job submission, driver, and client )

Reply:
yea, can be 3 ways including ray client.

Reply:
3 ways are fine too. I counted Driver and Client as the interactive way of running ray jobs above.